src/gui/base/CheckboxN.js

```diff
@@ -25,6 +26,9 @@ export class CheckboxN implements MComponent<CheckboxAttrs> {
 
 	view(vnode: Vnode<CheckboxAttrs>): Children {
 		const a = vnode.attrs
+		const helpLabel = a.disabled
+			? a.disabledTextId ? m("small.block.content-fg", lang.get(a.disabledTextId)) : []
```

Comment:
If disabled text has the same style as help label why do we need a separate attribute for it? Just pass in different text when `disabled` from the outside. I don't know if it's cleaner but it's simpler at least I think

Reply:
yeah that sounds better